### ISSUE TITLE
Implement fs.createReadStream

### ISSUE THREAD

**Issue by samccone** [oven-sh/bun#1207]
### Version

0.1.11

### Platform

Darwin sams-mac-studio.lan 21.6.0 Darwin Kernel Version 21.6.0: Sat Jun 18 17:07:22 PDT 2022; root:xnu-8020.140.41~1/RELEASE_ARM64_T6000 arm64

### What steps will reproduce the bug?

```
touch test.txt
echo "import * as fs from 'fs'; fs.createReadStream('test.txt');" > tmp.js
bun tmp.js
```

### How often does it reproduce? Is there a required condition?

100%

### What is the expected behavior?

does not raise a system error

### What do you see instead?

```
1 |                                                            │
2 | fs.createReadStream("test.txt");                           │
                             ^                                 │
error: SystemError                                             │
      at /Users/samccone/repos/deconstructed/tmp.js:1:26  
      ```

### Additional information

works in node.

**Comment by Jarred-Sumner** [comment]
It's not implemented yet

It's supposed to say "TODO" but it clearly is not saying that

**Comment by Electroid** [comment]
This was implemented in release `0.2.2`! 🎉
